python_modules/libraries/dagster-dbt/dagster_dbt/__init__.py

```diff
@@ -141,6 +144,19 @@
             ("dbt_test_op", "dagster_dbt.ops"),
         ]
     },
+    **{
+        value: (
+            module,
+            "0.24.0",
+            "Use the @dbt_assets decorator, DbtCliResource, and DagsterDbtTranslator instead. "
+            "To continue generating a dbt manifest at build time or run time, see the reference: "
```

Comment:
The reference is mainly relevant if they're using `load_assets_from_dbt_project`, right? If so, I think it's worth having different deprecation messages for the two APIs.

For migrating from `load_assets_from_dbt_manifest`, is there any other content we could link to?

Reply:
I've separated the two messages. I'll link the general content for creating the manifest at build time in the CI/CD system for `load_assets_from_dbt_manifest`.